data-catalog-policies-app/src/main/resources/application-itest.properties

```diff
@@ -11,4 +11,5 @@ spring.flyway.schemas=policy_schema
 spring.flyway.baseline-on-migrate=true
 
 # Endpoints
-datacatalog.codelist.purpose.url=http://35.201.118.102/backend/codelist/PURPOSE
\ No newline at end of file
+datacatalog.codelist.url=http://35.201.118.102/backend/codelist
```

Comment:
Can we try to avoid the use of IP address by using a relative path or Google Cloud secrets?